lib/wcc/contentful/services.rb

```diff
@@ -79,7 +79,8 @@ def preview_client
               preview_token: config.preview_token,
               space: config.space,
               default_locale: config.default_locale,
-              adapter: config.http_adapter
+              adapter: config.http_adapter,
+              environment: config.environment
```

Comment:
Have you tested this?  Does the preview API work with environments?

Reply:
Yeah, I am trying it out right now as we speak. It seems to work well. The URL that is being used:

```
effective_url=https://preview.contentful.com/spaces/asdfasdfasdfasdfasdfasdf/environments/review-for-qa/entries/987qa2w3gfoagwvloig?locale=%2A 
response_code=200 
return_code=ok 
total_time=0.39261
```